### PR TITLE
Multi-mission radioisotope thermoelectric generator

### PR DESCRIPTION
**Pull request by everythingeverywhere**
https://github.com/instructlab/taxonomy

If your PR is related to a contribution to the taxonomy, please, fill
out the following questionnaire. If not, replace this whole text and the
following questionnaire with whatever information is applicable to your PR.

**Describe the contribution to the taxonomy**

- ...
- ...
- ...

**Input given at the prompt**

```
   ...
```

**Response from the original model**

```
  ...
```

**Response from the fine-tuned model**

```
  ...
```

**Contribution checklist**

- [ ] The contribution was tested with `ilab generate`
- [ ] No errors or warnings were produced by `ilab generate`
- [ ] All [commits are signed off](https://github.com/instructlab/taxonomy/blob/main/CONTRIBUTING.md#legal) (DCO)
- [ ] The `qna.yaml` file contains at least 5 `seed_examples`
- [ ] The `qna.yaml` file was [linted](https://yamllint.com) and [prettified](https://onlineyamltools.com/prettify-yaml) ([yaml-validator](https://jsonformatter.org/yaml-validator) can do both)
- [ ] An `attribution.txt` file in the same folder as the `qna.yaml` file
- [ ] Content does not include PII or otherwise sensitive or confidential information
- [ ] Content does not include anything documented in the project's [Avoid these Topics](https://github.com/instructlab/taxonomy/blob/main/docs/SKILLS_GUIDE.md#avoid-these-topics) guidelines

### CODE REVIEW

**Comment by everythingeverywhere** [comment]
https://github.com/instructlab/taxonomy

**Comment by jjasghar** [comment]
Please fix this so it's in the correct taxonomy leaf node (directory).

File: attribution.txt
Comment (jjasghar) [inline attribution.txt:1]: Please put this file in the correct directory.

File: qna.yaml
Comment (jjasghar) [inline qna.yaml:1]: Please put this file in the correct directory.

File: qna.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+created_by: everythingeverywhere
+domain: astronomy_technology
+seed_examples:
+- answer: | a type of radioisotope thermoelectric generator (RTG) developed for NASA space missions[1] such as the Mars Science Laboratory (MSL), under the jurisdiction of the United States Department of Energy's Office of Space and Defense Power Systems within the Office of Nuclear Energy
```
Comment (jjasghar) [inline qna.yaml:4]: Thank you for your submission; taking a look at your `qna.yaml`, we noticed that you have some issues with the multiline `yaml` syntax.

Can you take a look at: https://yaml-multiline.info/ and fix up the file? Thanks!

**Review by jjasghar — CHANGES_REQUESTED** [review]
Please resolve these issues then we can run the linter.

**Comment by jjasghar** [comment]
Closing due to combined PR here:  #1142

**Pull request closed without merging**